src/components/TextInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Field, propTypes } from 'redux-form';
+import { TextField } from 'redux-form-material-ui';
+
+
+const TextInput = ({ name, type, placeholder, ...props }) => (
+  <Field
+    name={name}
+    component={TextField}
+    type={type}
```

Comment:
I can't find `type` in the [documentation](http://redux-form.com/6.8.0/docs/api/Field.md/). What is it for?